### PR TITLE
Check package version which should be upgrade to expected version

### PR DESCRIPTION
**Pull request by tinawang123**
Check package version which should be upgrade to expected version
Some jira features need update some packages' version, we need check package version has been updated after migration testing.

- Related ticket: https://progress.opensuse.org/issues/60305
- Verification run: 
https://openqa.nue.suse.com/tests/4001161#

### CODE REVIEW

File: tests/console/check_package_version.pm [hunk 1]
```diff
@@ -73,16 +94,11 @@ sub run {
...
-        foreach my $key (keys %package_s390x) {
-            my $pcks = cmp_packages($key, $package_s390x{$key}[0], $package_s390x{$key}[1]);
-        }
-    }
-
-    # jsc#5668:Replace init script of ebtables with systemd service file
-    zypper_call('in ebtables');
-    systemctl 'start ebtables';
-    systemctl 'is-active ebtables';
+    #if (get_var('ARCH') =~ /s390x/) {
```
Comment (lemon-suse) [inline tests/console/check_package_version.pm]: If no use, could you remove these lines.

File: tests/console/check_package_version.pm [hunk 2]
```diff
@@ -24,7 +24,7 @@ use testapi;
 use utils qw(systemctl zypper_call);
 use Mojo::Util 'trim';
 
-my %package = (
+my %package_12sp5 = (
```
Comment (lemon-suse) [inline tests/console/check_package_version.pm]: These packages are 12SP5 special?

Comment (tinawang123) [reply tests/console/check_package_version.pm]: > These packages are 12SP5 special?

Yes

Comment (coolgw) [inline tests/console/check_package_version.pm]: maybe we can remove 12sp5 part since project already finished.

Comment (tinawang123) [reply tests/console/check_package_version.pm]: If you think it is no value to keep it. I can remove them.

**Comment by lemon-suse** [comment]
Please don't forget to update the copyright date for tests/console/check_package_version.pm

**Pull request merged**